cognite/neat/utils/rdf_.py

```diff
@@ -10,25 +12,28 @@
 
 @overload
 def remove_namespace_from_uri(
-    *URI: URIRef | str,
+    URI: URIRef | str,
+    *,
     special_separator: str = "#_",
     validation: Literal["full", "prefix"] = "prefix",
 ) -> str: ...
 
 
 @overload
 def remove_namespace_from_uri(
-    *URI: tuple[URIRef | str, ...],
+    URI: SequenceNotStr[URIRef | str],
+    *,
     special_separator: str = "#_",
     validation: Literal["full", "prefix"] = "prefix",
-) -> tuple[str, ...]: ...
+) -> list[str]: ...
 
 
 def remove_namespace_from_uri(
-    *URI: URIRef | str | tuple[URIRef | str, ...],
+    URI: URIRef | str | SequenceNotStr[URIRef | str],
+    *,
     special_separator: str = "#_",
     validation: Literal["full", "prefix"] = "prefix",
-) -> tuple[str, ...] | str:
+) -> str | list[str]:
     """Removes namespace from URI
```

Comment:
The overload was not correctly implemented leading to some unnecessary casting. So fixed the implementation